native_client/definitions.mk

```diff
@@ -170,3 +173,36 @@ define copy_missing_libs
         done; \
     fi;
 endef
+
+SWIG_DIST_URL ?= 
+ifeq ($(findstring Linux,$(OS)),Linux)
+SWIG_DIST_URL := "https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.swig.linux.amd64.b5fea54d39832d1d132d7dd921b69c0c2c9d5118/artifacts/public/ds-swig.tar.gz"
+endif
+ifeq ($(findstring Darwin,$(OS)),Darwin)
+SWIG_DIST_URL := "https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.swig.darwin.amd64.b5fea54d39832d1d132d7dd921b69c0c2c9d5118/artifacts/public/ds-swig.tar.gz"
+endif
+ifeq ($(findstring _NT,$(OS)),_NT)
+SWIG_DIST_URL := "https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.swig.win.amd64.b5fea54d39832d1d132d7dd921b69c0c2c9d5118/artifacts/public/ds-swig.tar.gz"
+endif
```

Comment:
Should we add a catch all clause printing a message saying that a prebuilt archive is not available? Otherwise it's just going to explode with a weird message in the wget below.

Reply:
Good idea.